src/Runner.ts

```diff
@@ -118,6 +145,50 @@ function getPackages(dependencies: { [pkg: string]: string; }): string[] {
 }
 
 /**
+ * Generate an HTML page which represents the Runner properties
+ * @param param0 Properties from the Runner to be used to specify the document
```

Comment:
I'm not familiar with the param0 syntax. If that is correct, ignore me, and if not, then of course fix it.